webhook/resources/backup/mutator.go

```diff
@@ -65,13 +64,19 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 		metaLabels = map[string]string{}
 	}
 
-	//check snapshot name, if not exist, return error
+	getVolume := datastore.LazyGetter(func() (*longhorn.Volume, error) {
```

Comment:
What's the purpose of the LazyGetter?

Reply:
To reuse the volume CR we've got. With this cache, the mutator only need one GET operation for the volume CR if there are multiple mutation needed, including the backup volume, the backup target, the access mode, and the block size.

Reply:
What does `multiple mutation` mean here? The volume CR is got from local cache. It seems there is no too much benefit from the lazy get.